concourse/pipelines/autodelete-cloudfoundry.yml

```diff
@@ -29,6 +50,16 @@ jobs:
         trigger: true
       - get: bosh-secrets
       - get: paas-cf
+      - get: cf-secrets
+      - get: cf-manifest
+      - get: bosh-CA
+
+      - task: get-cf-cli-config
+        file: paas-cf/concourse/tasks/get-cf-cli-config.yml
+
+      - task: remove-healthcheck-app
+        file: paas-cf/concourse/tasks/remove-healthcheck-app.yml
+
```

Comment:
As I remember to delete a service you need to unbind an app. During unbinding the app is restarted so it takes time. Deletion should be quicker. I renamed the task name to `remove-healthcheck-db`.